week7/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import logoImg from "../assets/Linkbrary.png";
+import "./Header.css";
+import Folder from "./Folder.js";
+import { useMediaQuery } from "react-responsive";
+const profileAccount = ({ user }) => {
+  if (!user) {
+    return false;
+  }
+  const { email, profileImageSource } = user;
+  return (
+    <div className="profile-info">
+      <img className="logo" src={profileImageSource} alt="프로필 이미지" />
+      <div>{email}</div>
```

Comment:
`div`보다는 텍스트를 위한 태그  `p`태그를 사용하는게 더 적절합니다.

```suggestion
      <p>{email}</p>
```